unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/ProjectPublishHUD/Scripts/Projects/PublishMapView.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using DCL.Helpers;
+using DCL.Interface;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.UIElements;
+
+namespace DCL.Builder
+{
+    public class PublishMapView : MonoBehaviour
+    {
+        public event Action<Vector2Int> OnParcelClicked;
+
+        [Header("References")]
+        [SerializeField] internal ScrollRect scrollRect;
+        [SerializeField] RectTransform scrollRectContentTransform;
+
+        RectTransform minimapViewport;
+        Transform mapRendererMinimapParent;
+        Vector3 atlasOriginalPosition;
+
+        private bool isVisible = false;
+
+        void Start()
+        {
+            scrollRect.onValueChanged.AddListener((x) =>
+            {
+                Debug.Log("x " + x.x + "   y " + x.y);
+                if (isVisible)
+                    MapRenderer.i.atlas.UpdateCulling();
+            });
+
+            MapRenderer.OnParcelClicked += ParcelSelect;
+        }
+
+        private void OnDestroy() { MapRenderer.OnParcelClicked -= ParcelSelect; }
+
+        internal void UpdateOwnedLands()
+        {
+            List<Vector2Int> landsToHighlight = new List<Vector2Int>();
+            foreach (var land in DataStore.i.builderInWorld.landsWithAccess.Get())
+            {
+                foreach (Vector2Int landParcel in land.parcels)
+                {
+                    landsToHighlight.Add(landParcel);
+                }
+            }
+            MapRenderer.i.HighlightLandsInRed(landsToHighlight);
+        }
+
+        public void GoToCoords(Vector2Int coords)
+        {
+            //Reset scroll
+            scrollRectContentTransform.anchoredPosition = new Vector2(-1219, -1596);
```

Comment:
Could we move these `-1219, -1596` coords to a constant with a descriptive name?